packages/form-builder/addon/components/cfb-slug-input.js

```diff
@@ -30,6 +31,16 @@ export default class CfbSlugInputComponent extends Component {
 
   @action
   calculatePadding(element) {
+    // Ensures that the element is already visible in combined use
+    // with modal dialogs.
+    scheduleOnce(
+      "afterRender",
+      this,
+      this._calculatePadding.bind(this, element),
```

Comment:
To prevent `ember/no-incorrect-calls-with-inline-anonymous-functions` linting error.

Reply:
Just pass "_calculatePadding" - this will take care of bindings.

Reply:
To be precise:

```js
scheduleOnce("afterRender", this, "_calculatePadding", element)
```